### PR TITLE
added viz_options option to table_chart

### PR DESCRIPTION
**Pull request by rajeshkurup**

### CODE REVIEW

**Comment by rajeshkurup** [comment]
hi @keitwb, I'm from PayPal and we create SignalFX table charts using Terraform. Can you please review my PR and help to release my code? It is required for us to modify our charts. thanks!

**Comment by rajeshkurup** [comment]
@keitwb

**Comment by rajeshkurup** [comment]
@jrcamp Hi, can you please review my PR and help to release my code change? thanks!

File: signalfx/resource_signalfx_table_chart_test.go [hunk 1]
```diff
@@ -20,6 +20,13 @@ resource "signalfx_table_chart" "mychartTB" {
 	hide_timestamp = true
 	group_by = ["ClusterName"]
 
+	viz_options {
```
Comment (seonsfx) [inline signalfx/resource_signalfx_table_chart_test.go:23]: I understand that it is missing in some other places, but can we add some assertions to test these new `viz_options`s in `TestAccCreateUpdateTableChart` function?

**Review by seonsfx — CHANGES_REQUESTED** [review]
Thank you very much for making these changes. This looks good to me mostly, and a couple of new assertions in the test case would make this even more perfect I think.

File: signalfx/resource_signalfx_table_chart_test.go (hunk 1 above)
Comment (rajeshkurup) [reply signalfx/resource_signalfx_table_chart_test.go:23]: Hi @seonsfx , your comment has been fixed in my new commit - please review.

**Comment by rajeshkurup** [comment]
> Thank you very much for making these changes. This looks good to me mostly, and a couple of new assertions in the test case would make this even more perfect I think.

You're welcome! I've fixed your comment in my new commit. Please review. Appreciate your comments and time taken for reviewing my code!

**Review by seonsfx — APPROVED** [review]

**Comment by rajeshkurup** [comment]
Hi @seonsfx , can you please merge this change and release new provider version? we're currently blocked to release our table charts due to this issue. Appreciate your help!

**Pull request merged**